src/lib/sidebar/Instructions.svelte

```diff
@@ -5,61 +5,32 @@
 </script>
 
 <Modal title="Instructions" bind:open>
-  <p>For each intervention:</p>
-
-  <ol>
-    <li>
-      Click one of the draw icons to the top right of the map (the icons allow
-      you to draw points, lines, polygons or routes)
-    </li>
-    <li>
-      Draw the intervention on the map, clicking the original point or 'Finish
-      snapping' for the Polygon and Route tools respectively
-    </li>
-    <li>
-      Fill in the details that appear (Intervention type, name, description)
-    </li>
-    <li>
-      Click the "Save" button or "Delete" button to save or remove the
-      intervention
-    </li>
-  </ol>
-
-  <p>
-    Repeat the steps above for each substantial infrastructure intervention in
-    the scheme.
-  </p>
-
+  <h2>Drawing objects on the map</h2>
   <p>
-    To make further edits to attribute data, or to delete an intervention, click
-    on its name in the left hand panel or its geometry in the map.
+    Use the tools on right to sketch on the map. Each tool has a <b>Help</b> button
+    or instructions.
   </p>
-
   <p>
-    Click the "Export to GeoJSON" button to download the dataset representing
-    the scheme, which can be made up of one or more intervention, as a GeoJSON
-    file. After saving the file, you can delete the interventions, change the
-    scheme name, and start a new set of interventions representing a new scheme.
+    You can modify the object on the map using <b>Edit geometry</b> and clicking
+    something.
   </p>
 
+  <h2>Filling out data</h2>
   <p>
-    To load a GeoJSON file that you have already saved from the tool, click the
-    "Load from GeoJSON" button and select the file.
+    While you're in <b>Edit attributes</b> mode, you can select an object to describe
```

Comment:
maybe a slight rephrase to `you can select an object on the left to describe it` would  be slightly clearer